bundler/lib/bundler/cli/outdated.rb

```diff
@@ -54,7 +54,7 @@ def run
       end
 
       if options[:parseable]
-        Bundler.ui.silence(&definition_resolution)
+        Bundler.ui.with_output_stream(:stderr, &definition_resolution)
```

Comment:
Would it be worth using a higher-level name? Such as

```suggestion
        Bundler.ui.progress(&definition_resolution)
```

I don't see `with_output_stream` being used for anything else than stderr progress.

Reply:
Given these three suggestions, I think I like `progress` the best. I'll make that a public method on `UI::Shell` and make `with_output_stream` private, like this.

```ruby
def progress(&block)
  with_output_stream(:stderr, &block)
end

private

def with_output_stream(symbol)
  original = output_stream
  self.output_stream = symbol
  yield
ensure
  @output_stream = original
end
```
Does that sound good?

Reply:
Sounds good to me!

Reply:
Done ✔️